samples/graph-appcatalog-lifecycle/csharp/README.md

```diff
@@ -44,6 +44,65 @@ This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalo
 
     ```bash
     ngrok http -host-header=rewrite 3978
+    ```
+    
+## Register Azure AD application
+Register one Azure AD application in your tenant's directory: for the bot and tab app authentication.
+
+1.  Log in to the Azure portal from your subscription, and go to the "App registrations" blade  [here](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps). Ensure that you use a tenant where admin consent for API permissions can be provided.
+
+2.  Click on "New registration", and create an Azure AD application.
+
+3.  **Name:**  The name of your Teams app - if you are following the template for a default deployment, we recommend "App catalog lifecycle".
+
+4.  **Supported account types:**  Select "Accounts in any organizational directory"
+
+5.  Leave the "Redirect URL" field blank.   
+
+6.  Click on the "Register" button.
+
+7.  When the app is registered, you'll be taken to the app's "Overview" page. Copy the  **Application (client) ID**; we will need it later. Verify that the "Supported account types" is set to  **Multiple organizations**.
+
+8.  On the side rail in the Manage section, navigate to the "Certificates & secrets" section. In the Client secrets section, click on "+ New client secret". Add a description for the secret and select Expires as "Never". Click "Add".
+
+9.  Once the client secret is created, copy its  **Value**, please take a note of the secret as it will be required later.
+
+
+At this point you have 3 unique values:
+-   Application (client) ID which will be later used during Azure bot creation
+-   Client secret for the bot which will be later used during Azure bot creation
+-   Directory (tenant) ID
+
+
+We recommend that you copy these values into a text file, using an application like Notepad. We will need these values later.
+
+10.  Under left menu, select  **Authentication**  under  **Manage**  section.
+
+11. Select 'Accounts in any organizational directory (Any Azure AD directory - Multitenant)' under Supported account types and click "+Add a platform".
+
+12.  On the flyout menu, Select "Web"    
+
+13.  Add  `https://token.botframework.com/.auth/web/redirect`  under Redirect URLs and click Configure button. 
+
+14.  Once the flyout menu close, scroll bottom to section 'Implicit Grant' and select check boxes "Access tokens" and "ID tokens" and click "Save" at the top bar.
+
+15.  Under left menu, navigate to  **API Permissions**, and make sure to add the following permissions of Microsoft Graph API > Delegated permissions:
+-    AppCatalog.ReadWrite.All
+-  AppCatalog.Submit
+
+Click on �Add Permissions� to commit your changes.
```

Comment:
Why special char is showing in this file?

Reply:
Updated